core/config/docs/core.toml

```
@@ -601,3 +601,26 @@ TLSCertPath = '/path/to/cert.pem' # Example
 [Tracing.Attributes]
 # env is an example user specified key-value pair
 env = 'test' # Example
+
+[Mercury]
```

Would it make sense to name this `LLO`? or will those settings be separate?
```suggestion
[LLO]
```

---

That's fine I guess, but we'll have to figure out how to navigate those breaking config changes carefully.